lib/webtask.js

```diff
@@ -191,22 +201,22 @@ Webtask.prototype.createCronJob = function (options, cb) {
 };
 
 
-/**
- * Inspect an existing webtask to optionally get code and/or secrets
- *
- * @param {Object} options - Options for inspecting the webtask.
- * @param {Boolean} [options.decrypt] - Decrypt the webtask's secrets.
- * @param {Boolean} [options.fetch_code] - Fetch the code associated with the webtask.
- * @param {Function} [cb] - Optional callback function for node-style callbacks.
- * @returns {Promise} A Promise that will be fulfilled with the result of inspecting the token.
- */
-Webtask.prototype.inspect = function (options, cb) {
-    options.token = this.token;
+// /**
```

Comment:
Why was this method commented out? Wouldn't that result in a major breaking change?